packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -1298,9 +1298,23 @@ export const getUserChallenges = (
 export const getPassingRecord = (
   userId: string | number,
   accessToken: string | undefined,
-): Promise<PassingRecord> => {
+): Promise<PassingRecord | null> => {
```

Comment:
This will return 404 if you haven't taken the quiz, so update it to properly reflect that